topics/ethereum/index.md

```diff
@@ -0,0 +1,13 @@
+---
+created_by: Vitalik Buterin
+display_name: Ethereum
+logo: ethereum.png
+released: July 30, 2015
+short_description: Ethereum is a distributed public blockchain network.
+topic: ethereum
+related: blockchain, cryptocurrency
+url: https://www.ethereum.org/
+github_url: https://github.com/ethereum
+wikipedia_url: https://en.wikipedia.org/wiki/Ethereum
+---
+Ethereum is a decentralized platform that runs contracts based applications without any possibility of downtime, censorship, fraud or third-party interference. Ethereum blockchain focuses on running the programming code of any decentralized application. 
```

Comment:
- `contracts based` -> `contracts-based` or `contract-based`
- is it `running the programming code`, `writing the code` or `maintaining the code`?

Reply:
Is it wrong to use "running the programming code"?

Reply:
@himanshu1603 `programming code` feels a bit weird/redundant to me; I think most people in the context of GitHub will assume `code` == `programming code`. What do you think?

Reply:
what about 'running the programs'?

Reply:
@himanshu1603 In that case it'd just be `Ethereum blockchain focuses on running any decentralized applications.` Either that or my suggestion which was `Ethereum blockchain focuses on running the code of any decentralized application.`